internal/batches/workspace/git.go

```diff
@@ -25,9 +25,9 @@ func runGitCmd(ctx context.Context, dir string, args ...string) ([]byte, error)
 		"GIT_COMMITTER_EMAIL=batch-changes@sourcegraph.com",
 	}
 	cmd.Dir = dir
-	out, err := cmd.CombinedOutput()
+	out, err := cmd.Output()
```

Comment:
TODO: Why is this suddenly happening

Reply:
I am seeing 

```
fatal: could not read Password for 'http://executor@localhost:3080': terminal prompts disabled
fatal: could not read Password for 'http://executor@localhost:3080': terminal prompts disabled
M  client/client-api/README.md
diff --git client/client-api/README.md client/client-api/README.md
index 7462bfb..e2eb036 100644
--- client/client-api/README.md
+++ client/client-api/README.md
@@ -1,3 +1,4 @@
 # Client API
 
 Sourcegraph Client API
+Hello World
```
for git status.. why would it ask for credentials for git status 🤔